src/app/shared/components/timer/timer.component.html

```diff
@@ -0,0 +1,10 @@
+<p>This thing work</p>
+
+<div class="timer">
+  <span class="time">{{ time.length < 3 ? 0 : (time | slice : 0 : -2) }}</span>
+  <span class="time">{{ time | slice : -2 }}</span>
+</div>
+
+<button (click)="onStart()" [disabled]="clock">Start</button>
+<button (click)="onStop()" [disabled]="!clock">Stop</button>
+<button (click)="onReset()" [disabled]="!clock && time === '0'">Reset</button>
```

Comment:
for disabling buttons you can emit `stop$` with `Boolean` and check with [async pipe]{https://angular.io/api/common/AsyncPipe}

Reply:
Always false.

Reply:
In code 
![image](https://user-images.githubusercontent.com/119358804/205671848-da5ebc5a-0d01-407d-beb9-8d1fb4d26050.png)
In web
![image](https://user-images.githubusercontent.com/119358804/205672006-19aa0a80-a18e-4d95-9183-e4006023502b.png)

Reply:
Have you try `this.stop$.next(false)`